drivers/flash/flash_ll_nrfxlib.c

```diff
@@ -0,0 +1,337 @@
+/*
```

Comment:
Similar  as https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/687#discussion_r278570181
I think providing dedicated flash driver is bad design decision.
Only think which was really needed was a API for request time-window chain for perform flash operation as it is already implemented by the flash driver. If it would be required - then refined zephyr API for that.

What is missing in nrf_flash_driver that this one is needed?

If proprietary BLE controller requires a priority access to the flash - we would also introduce solution for this.

Reply:
We had a meeting discussing this some time back and came to that this was the best solution with the given constraints.